EKrut-Server/src/ekrut/server/db/ReportDAO.java

```diff
@@ -591,13 +657,19 @@ public boolean createOrderReport(Report report) {
 		
 		con.beginTransaction();
 		
-		PreparedStatement ps1 = con.getPreparedStatement("INSERT INTO orderMonthlySales"
-														+ "(reportID,itemName,totalOrders) " +
-                    									"VALUES(?,?,?)");
+		PreparedStatement ps1 = con.getPreparedStatement("INSERT INTO topSellers"
+								+ "(reportID,itemName,totalSales) " +
```

Comment:
You're missing a space between the table name and the list of columns in all of these. Not sure if the space is necessary but better to fix this.